circuits/ts/__tests__/CeremonyParams.test.ts

```diff
@@ -11,31 +12,25 @@ import { generateRandomIndex, getSignal, circomkitInstance } from "./utils/utils
 describe("Ceremony param tests", () => {
   const params = {
     // processMessages and Tally
-    stateTreeDepth: 6,
-    // processMessages
-    messageTreeDepth: 9,
-    // processMessages
-    messageBatchTreeDepth: 2,
+    stateTreeDepth: 10,
```

Comment:
let's use 14 here ( 2 ** 14 is around the same signups as 5 ** 6 which is what we use in production)